lms/static/scripts/frontend_apps/components/ValidationMessage.tsx

```diff
@@ -1,36 +1,33 @@
 import classnames from 'classnames';
 import { useEffect, useState } from 'preact/hooks';
 
-/**
- * @typedef ValidationMessageProps
- * @prop {string} message - Error message text
- * @prop {boolean} [open] - Should this be open or closed
```

Comment:
Using the terms  "open" and "closed" to refer to a "message" raised my eyebrows. Reading through the code it looks like what it means is that the message can be dismissed by clicking on it. I have reworded the descriptions accordingly, but not renamed the props here. This whole component might merit a revamp at some point.

Reply:
Yeah, I'm with you. Refactoring this is very much on my radar.